tests/test_talonsrx.py

```diff
@@ -161,9 +161,9 @@ def test_basemotorcontroller_configForwardSoftLimitThreshold(talon, cdata):
     assert cdata["soft_limit_for"] == 1
 
 
-@pytest.mark.xfail(raises=NotImplementedError)
-def test_basemotorcontroller_configGetCustomParam(talon):
-    talon.configGetCustomParam(1, 2)
+def test_basemotorcontroller_configGetCustomParam(talon, cdata):
+    param = talon.configGetCustomParam(1, 2)
+    assert param == cdata["custom_param_1"]
```

Comment:
This is… not quite what I was after. This should test that if you change the hal_data dict, the return value is what you expect (and not what is currently in the dict - we want to make sure the method doesn't write to it!).